mitodl/bootcamp-ecommerce
# Added command to migrate applications from one run to another

#### What are the relevant tickets?
Closes #1073 

#### What's this PR do?
Adds a management command to migrate applications from one bootcamp run to another

#### How should this be manually tested?
1. Find or create a bootcamp that has at least 2 runs
2. Create an application for an earlier run of that bootcamp and set it to complete
3. Run the `migrate_applications` command, specifying the correct `--from-run` and `--to-run` (you can use the run id or title)

This should create a new application in the 'to' run for your user. It should be in the `AWAITING_PAYMENT` state, it should have the same resume info, the same submissions, and those submissions should be approved. The command should be idempotent, so running it again should not change any data.

You can also test the `--users` parameter by adding a comma-separated string of emails to limit the migration to specific users
